packages/styles/scss/patterns/cards-with-images/index.scss

```diff
@@ -0,0 +1,6 @@
+@import '../../globals/imports';
+@import './cards-with-images';
+@import './cards-with-images-group';
+
+@include cards-with-images;
+@include cards-with-images-group;
```

Comment:
Are you also importing this file into `ibm-dotcom-styles.scss`?

Reply:
No. I will import here also